packages/renderer/src/lib/ui/SearchInput.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+import Input from './Input.svelte';
+
+export let title: string;
+export let searchTerm: string = '';
+</script>
+
+<Input
+  class="{$$props.class || ''}"
+  id="search-{title}"
+  name="search-{title}"
+  placeholder="Search {title}..."
+  bind:value="{searchTerm}"
+  aria-label="search"
```

Comment:
I think here we're missing title as well no ?

Reply:
Updated aria-label (for search and password).